networkx/algorithms/cuts.py

```diff
@@ -359,40 +359,58 @@ def node_expansion(G, S):
     return len(neighborhood) / len(S)
 
 
-# TODO What is the generalization to two arguments, S and T? Does the
-# denominator become `min(len(S), len(T))`?
 @nx._dispatchable
 def boundary_expansion(G, S):
     """Returns the boundary expansion of the set `S`.
 
-    The *boundary expansion* is the quotient of the size
-    of the node boundary and the cardinality of *S*. [1]
+    The *boundary expansion* of a set `S` is the ratio between the size
+    of its node boundary and the cardinality of the set itself. [1]
 
     Parameters
     ----------
     G : NetworkX graph
+        The input graph.
 
     S : collection
         A collection of nodes in `G`.
 
     Returns
     -------
     number
-        The boundary expansion of the set `S`.
+        The boundary expansion ratio: size of node boundary / size of S
+
+    Examples
+    --------
+    >>> G = nx.Graph([(0, 1), (1, 2), (2, 3), (3, 0)])
+    >>> S = {0, 1}
+    >>> nx.boundary_expansion(G, S)
+    0.5  # Node boundary is {2, 3} (size 2), divided by |S|=4
```

Comment:
doctests required the outputs to match the expected output *exactly*, as in character-per-character. Putting comments like this will cause tests to fail - instead try something like:

```suggestion
    
    Node boundary is {2, 3} (size 2), divided by |S| = 4:
    
    >>> nx.boundary_expansion(G, S)
    0.5
```

(though I'm confused - isn't |S| = 2? D you mean |S| + |node boundary|?)